core/templates/components/skill-selector/questions-list-select-skill-modal.controller.spec.ts

```diff
@@ -44,13 +44,14 @@ describe('Questions List Select Skill Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
+  it('should initialize correctly $scope properties after controller' +
+    ' initialization', function() {
     expect($scope.skillSummaries).toEqual(sortedSkillSummaries);
     expect($scope.selectedSkillId).toBe(null);
     expect($scope.countOfSkillsToPrioritize).toBe(skillsInSameTopicCount);
   });
 
-  it('should check properties set after controller is initialized', function() {
+  it('should save selected skill and close modal when saving', function() {
```

Comment:
Again, I don't know what "saving" is.. Try to see what calls this save() function.

Reply:
Done.